packages/asc-ui/src/components/TopTaskButton/TopTaskButtonStyle.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from '@datapunt/asc-core'
+import { themeSpacing, themeColor } from '../../utils'
+
+import IconStyle from '../Icon/IconStyle'
+
+export interface Props {
+  icon: React.ElementType
+  title?: string
+  content?: string
+  onDarkBackground?: boolean
+  width: string
+  className: string
+}
+
+const defaultProps = {
+  onDarkBackground: false,
+  width: '220px',
+}
+
+export default styled.div<Props>`
+  width: ${({ width = defaultProps.width }) => width};
+  padding: ${themeSpacing(2)} ${themeSpacing(4)};
+  cursor: pointer;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 23px;
+  text-decoration: none;
+  color: ${themeColor('tint', 'level7')};
+  box-shadow: 2px 2px 0 0
+    ${({ onDarkBackground = defaultProps.onDarkBackground }) =>
+      onDarkBackground
+        ? themeColor('tint', 'level7')
+        : themeColor('tint', 'level3')};
+  background-color: ${themeColor('tint', 'level2')};
+
+  :hover {
+    box-shadow: 2px 2px ${themeColor('secondary', 'main')};
+    color: ${themeColor('secondary', 'main')};
+    text-decoration: underline;
+  }
+
+  ${IconStyle} {
+    margin: 0;
+    padding: 0 0 10px 1px;
```

Comment:
Why the 1px? And can we use themeSpacing here?

Reply:
no it needs to be 1 pixel

Reply:
Yes but why? :P